From 5c7a5ad8398780971aebeb35eb77cc3a56c4d293 Mon Sep 17 00:00:00 2001
From: Gerd Hoffmann <kraxel@redhat.com>
Date: Thu, 21 Jul 2011 16:43:52 -0300
Subject: [RHEL6 qemu-kvm PATCH 14/65] usb-ehci: drop unused num-ports state member

RH-Author: Gerd Hoffmann <kraxel@redhat.com>
Message-id: <1311266648-1179-15-git-send-email-kraxel@redhat.com>
Patchwork-id: 30138
O-Subject: [RHEL-6.2 kvm PATCH 14/30] usb-ehci: drop unused num-ports state member
Bugzilla: 723858 723863
RH-Acked-by: Hans de Goede <hdegoede@redhat.com>
RH-Acked-by: Jes Sorensen <Jes.Sorensen@redhat.com>
RH-Acked-by: Alex Williamson <alex.williamson@redhat.com>

From: Hans de Goede <hdegoede@redhat.com>

Signed-off-by: Hans de Goede <hdegoede@redhat.com>
Signed-off-by: Gerd Hoffmann <kraxel@redhat.com>
(cherry picked from commit 053f9fcfb1d84ab2f1d8fbd608d5e31aae58cbb7)
---
 hw/usb-ehci.c |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

Signed-off-by: Eduardo Habkost <ehabkost@redhat.com>
---
 hw/usb-ehci.c |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/hw/usb-ehci.c b/hw/usb-ehci.c
index db50be0..97ab90c 100644
--- a/hw/usb-ehci.c
+++ b/hw/usb-ehci.c
@@ -372,7 +372,6 @@ struct EHCIState {
     qemu_irq irq;
     target_phys_addr_t mem_base;
     int mem;
-    int num_ports;
 
     /* properties */
     uint32_t freq;
-- 
1.7.3.2

